uint16 num_x       # The number of checkerboard corners in the x direction
uint16 num_y       # The number of checkerboard corners in the y direction
float64 pixel_tol # The max difference in sensed pixel location of a corner across a set of camera frames